packages/core/admin/admin/src/content-manager/pages/ListView/components/tests/AutoCloneFailureModal.test.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+
+import userEvent from '@testing-library/user-event';
+import { render as renderRTL, screen } from '@tests/utils';
+import { Route } from 'react-router-dom';
+
+import { AutoCloneFailureModal } from '../AutoCloneFailureModal';
+
+import type { Location } from 'history';
+
+const user = userEvent.setup();
+
+let testLocation: Location = null!;
+
+const render = (props: React.ComponentProps<typeof AutoCloneFailureModal>) =>
+  renderRTL(<AutoCloneFailureModal {...props} />, {
+    renderOptions: {
+      wrapper({ children }) {
+        return (
+          <>
+            {children}
+            <Route
+              path="*"
+              render={({ location }) => {
+                testLocation = location;
+
+                return null;
+              }}
+            />
+          </>
+        );
+      },
+    },
+    initialEntries: ['/content-manager/collection-types/api::model.model?plugins[i18n][locale]=en'],
+  });
+
+describe('AutoCloneFailureModal', () => {
+  it('renders nothing if there is no entryId', () => {
+    render({ entryId: null, onClose: jest.fn(), prohibitedFields: [], pluginQueryParams: '' });
+
+    expect(screen.queryByText(/duplicate/i)).not.toBeInTheDocument();
+  });
+
+  it('toggles the modal', async () => {
+    const onClose = jest.fn();
+    render({ entryId: 1, onClose, prohibitedFields: [], pluginQueryParams: '' });
+
+    await user.click(screen.getByRole('button', { name: /cancel/i }));
+    expect(onClose).toHaveBeenCalledTimes(1);
+  });
+
+  it('shows the fields that prevent duplication', async () => {
+    render({
+      entryId: 1,
+      onClose: jest.fn(),
+      prohibitedFields: [
+        [['dynZoneAttrName', 'Unique Component', 'componentAttrName', 'text'], 'unique'],
+        [['oneToOneRelation'], 'relation'],
```

Comment:
IMO we should also test that these field paths show as that's quite important to the UI :)

Reply:
yup, definitely forgot that part